src/transformers/models/opt/modeling_opt.py

```diff
@@ -398,7 +398,6 @@ class OPTPreTrainedModel(PreTrainedModel):
     config_class = OPTConfig
     base_model_prefix = "model"
     supports_gradient_checkpointing = True
-    _no_split_modules = ["OPTDecoderLayer"]
```

Comment:
the embedding code is directly inside `OPTDecoder`, and if this is in `_no_split_modules`, basically the whole model is not split

Reply:
This is somehow undesired change - we are trying to avoid test failure for edge cases (super tiny modes). Mabye we should keep `_no_split_modules` and just skip the test for this model

Reply:
The DecoderLayer has a cross-connection and needs to be here.